.github/workflows/build.yml

```diff
@@ -296,6 +302,25 @@ jobs:
           cp -R 'tmp.package/Assets/TextMesh Pro' Assets/
           rm -rf tmp.plugin tmp.package
 
+
+      - name: Manually delete Zapbox package
+        if: matrix.vrsdk != 'Zapbox'
+        run: |
+          # Android builds fail on if Zapbox package is present
+          cat Packages/manifest.json | jq 'del( .dependencies ["com.zappar.xr.zapbox"] )' | tee Packages/manifest.json.new
```

Comment:
props for using `tee` for debugging purposes, as opposed to just `> foo.new`

Reply:
I just learned from best! 😉